src/server/examples/examples_test.go

```diff
@@ -73,3 +74,156 @@ func TestExampleTensorFlow(t *testing.T) {
 	require.NoError(t, c.DeleteRepo("GoT_train", false))
 	require.NoError(t, c.DeleteRepo("GoT_scripts", false))
 }
+
+func TestFruitStand(t *testing.T) {
+
+	if testing.Short() {
+		t.Skip("Skipping integration tests in short mode")
+	}
+
+	c := getPachClient(t)
+	t.Parallel()
+
+	require.NoError(t, c.CreateRepo("data"))
+	repoInfos, err := c.ListRepo(nil)
+	require.NoError(t, err)
+	var repoNames []interface{}
+	for _, repoInfo := range repoInfos {
+		repoNames = append(repoNames, repoInfo.Repo.Name)
+	}
+	require.OneOfEquals(t, "data", repoNames)
+
+	cmd := exec.Command(
+		"pachctl",
+		"put-file",
+		"data",
+		"master",
+		"sales",
+		"-c",
+		"-f",
+		"https://raw.githubusercontent.com/pachyderm/pachyderm/master/doc/examples/fruit_stand/set1.txt",
```

Comment:
I see. We can make that change if you'd like. (You're right it is a bit annoying but only if you're modifying this test, or the corresponding JSON file. In this case CI will do the right thing / or you can push to your branch as you develop if you want to run tests locally )

My goal with this approach was to make the test as similar to what a user will actually type. We don't currently have CLI test coverage and so I don't think we have coverage for `PutFile` from a file or URL.

[But we do have an issue for CLI coverage](https://github.com/pachyderm/pachyderm/issues/710).

So, I'll leave it up to you. Would you like me to change it to reading the file locally?